integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -70,18 +70,25 @@ def get_hix_context(self):
             language__slug__in=settings.TEXTLAB_API_LANGUAGES, page__in=hix_pages
         ).distinct("page_id", "language_id")
 
+        # Get all translations with empty content
+        hix_translations_with_empty_content = [
+            pt for pt in hix_translations if not pt.content
```

Comment:
```suggestion
            pt for pt in hix_translations if not pt.content.strip()
```
In the HIX signals, we also count a bunch of whitespaces as "empty"